install-deps.js

```diff
@@ -0,0 +1,37 @@
+const { exec } = require("child_process");
+const { promisify } = require("util");
+const asyncExec = promisify(exec);
+
+const workspaces = [
+  "american-british-translator",
+  "anonymous-message-board",
+  "build-a-pinterest-clone",
+  "chart-the-stock-market",
+  "exercise-tracker",
+  "file-metadata-microservice",
+  "forum-proxy",
+  "image-search-abstraction-layer",
+  "issue-tracker",
+  "manage-a-book-trading-club",
+  "metric-imperial-converter",
+  // "nightlife-coordination-app" - uncomment when this is ported,
+  "p2p-video-chat-application",
+  "personal-library",
+  "request-header-parser-microservice",
+  "secure-real-time-multiplayer-game",
+  "stock-price-checker",
+  "stock-price-checker-proxy",
+  "sudoku-solver",
+  "timestamp-microservice",
+  "twitch-proxy",
+  "url-shortener-microservice",
+  "voting-app",
+  "weather-proxy",
+];
+
+(async () => {
+  for (const workspace of workspaces) {
+    console.log(`Installing: ${workspace}`);
+    await asyncExec(`cd ${workspace} && npm ci`);
+  }
+})();
```

Comment:
```suggestion
for (const workspace of workspaces) {
  console.log(`Installing: ${workspace}`);
  asyncExec(`cd ${workspace} && npm ci`);
}
```
Installing them sequentially is a tad slow, could we do this instead?

Reply:
My only concern with that would be by not waiting for the install, we hit a race condition where it fires off these child processes and then attempts to test a project before it's done installing.

Reply:
It should be fine.  Those are all child processes triggered by the initial `npm ci` script, so that will wait for them before terminating.  The test script can't start until that's done.